doc/sphinx/source/drivers_doc.rst

```diff
@@ -0,0 +1,19 @@
+******************
+no-OS drivers list
+******************
+
+The following sections contain code documentation for ADI no-OS drivers.
+
+ACCELEROMETERS
+==============
+.. toctree::
+   :maxdepth: 1
+
+   drivers/adxl355
+
+ADC
```

Comment:
Just wondering, if we use ACCELEROMETERS, maybe we should switch to the Analog to Digital Converters, instead of ADC

Reply:
Sure, can do that.